vendor/opam/src/format/opamFilter.mli

```diff
@@ -50,6 +51,10 @@ val fold_down_left: ('a -> filter -> 'a) -> 'a -> filter -> 'a
 (** Maps on all nodes of a filter, bottom-up *)
 val map_up: (filter -> filter) -> filter -> filter
 
+(** Regex matching string interpolation syntax (["%%"], ["%{xxx}%"], or
+    ["%{xxx"] if unclosed) *)
+val string_interp_regex : Re.re
```

Comment:
Could you create a PR to OPAM to upstream this if possible?

Reply:
https://github.com/ocaml/opam/pull/5633